js/templates/pageNav.html

```diff
@@ -30,15 +30,15 @@
       <div>
         <div class="flexVCent box">
           <a class="iconBtn ion-android-notifications js-navNotifBtn"></a>
-          <a id="AvatarBtn" class="discSm border clrBr js-navListBtn navListBtn"
+          <a id="AvatarBtn" class="discSm border clrBr2 clrSh1 js-navListBtn navListBtn"
           <% if (ob.avatarHash) { %>
           //this will be updated to the real image parameter
           style="background-image: url(<%= ob.getServerUrl(`ipfs/${ob.avatarHash}`) %>), url('../imgs/defaultAvatar.png')"
           <% } else { %>
             style="background-image: url('../imgs/defaultAvatar.png')"
           <% } %>
           ></a>
-          <nav class="navList listBox clrS clrSh1 foldDown js-navList js-navPopMenu">
+          <nav class="navList listBox clrP clrSh1 clrBr foldDown js-navList js-navPopMenu">
```

Comment:
The v1 is pretty hacky, it hides the menu behind the top bar and moves it up, which creates bugs where it peaks through if we're not super careful.

The clipping path is pretty reliable, it'll just need some tweaking to make it wider to account for the shadow etc.

If that doesn't work for some reason, I'll come up with something else.